decentraland/unity-renderer
# feat: (sdk7) move pointer event results into multiple entities

## What does this PR change?
 This is pending optimization for the new SDK7 release, please read https://github.com/decentraland/js-sdk-toolchain/pull/403

## How to test the changes?
1. Go to the playground with the next query params added: https://playground.decentraland.org/?&sdk-branch=feat/retrieve-command-from-multiple-entities&explorer-branch=feat/pointer-event-result-into-multiple-entities
2. When all is ready, click on Run and the scene with the cube spawner should work. (Press E pointing to the main cube)
3. Here is the reference without parameter: https://playground.decentraland.org/
